Resources/Prototypes/Entities/Clothing/Ears/headsets.yml

```diff
@@ -139,115 +159,88 @@
   name: medical research headset
   description: A headset that is a result of the mating between medical and science.
   components:
-  - type: Headset
-    channels:
-    - Common
-    - Medical
-    - Science
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyMedicalScience
   - type: Sprite
     sprite: Clothing/Ears/Headsets/medicalscience.rsi
   - type: Clothing
     sprite: Clothing/Ears/Headsets/medicalscience.rsi
 
 - type: entity
   parent: ClothingHeadset
-  id: ClothingHeadsetMining
-  name: mining headset
-  description: Headset used by shaft miners.
-  components:
-  - type: Headset
-    channels:
-      - Common
-      - Supply
-  - type: Sprite
-    sprite: Clothing/Ears/Headsets/mining.rsi
-  - type: Clothing
-    sprite: Clothing/Ears/Headsets/mining.rsi
-
-- type: entity
-  parent: ClothingHeadset
-  id: ClothingHeadsetRobotics
-  name: robotics headset
-  description: Made specifically for the roboticists, who cannot decide between departments.
+  id: ClothingHeadsetSecurity
+  name: security headset
+  description: This is used by your elite security force.
   components:
-  - type: Headset
-    channels:
-      - Common
-      - Science
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeySecurity
   - type: Sprite
-    sprite: Clothing/Ears/Headsets/robotics.rsi
+    sprite: Clothing/Ears/Headsets/security.rsi
   - type: Clothing
-    sprite: Clothing/Ears/Headsets/robotics.rsi
+    sprite: Clothing/Ears/Headsets/security.rsi
```

Comment:
We already have a security headset, why did you replace the robotics headset with a copy of it?